boards/arm/stm32/stm3240g-eval/configs/telnetd/defconfig

```diff
@@ -26,17 +26,16 @@ CONFIG_EXAMPLES_TELNETD_DAEMONPRIO=128
 CONFIG_EXAMPLES_TELNETD_NOMAC=y
 CONFIG_HAVE_CXX=y
 CONFIG_HAVE_CXXINITIALIZE=y
-CONFIG_HOST_WINDOWS=y
```

Comment:
How removal of host windows flag is related this the described solution?

Reply:
> How removal of host windows flag is related this the described solution?

If I don't remove CONFIG_HOST_WINDOWS, CI will fail because defconfig is not pure.